src/sentry/integrations/bitbucket/uninstalled.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import absolute_import
+
+
+from django.views.decorators.csrf import csrf_exempt
+
+from sentry.api.base import Endpoint
+from sentry.constants import ObjectStatus
+from sentry.models import Repository
+from sentry.integrations.atlassian_connect import AtlassianConnectValidationError, get_integration_from_jwt
+
+
+class BitbucketUninstalledEndpoint(Endpoint):
+    authentication_classes = ()
+    permission_classes = ()
+
+    @csrf_exempt
+    def dispatch(self, request, *args, **kwargs):
+        return super(BitbucketUninstalledEndpoint, self).dispatch(request, *args, **kwargs)
+
+    def post(self, request, *args, **kwargs):
+        try:
+            token = request.META['HTTP_AUTHORIZATION'].split(' ', 1)[1]
+        except (KeyError, IndexError):
+            return self.respond(status=400)
+
+        try:
+            integration = get_integration_from_jwt(
+                token, request.path, 'bitbucket', request.GET, method='POST'
+            )
+        except AtlassianConnectValidationError:
+            return self.respond(status=400)
+
+        integration.update(status=ObjectStatus.DISABLED)
```

Comment:
Would we not want to just delete here considering it'll just sit in the database anyway?

Reply:
If the `Integration` is disabled, I'd argue that all `OrganizationIntegration`s should also be disabled.

Reply:
IMO `OrganizationIntegration` can stay enabled, since by relinking the integration the OrganizationIntegration will work again.

I'm actually not 100% sure what the usecase for adding a status on the `OrganizationIntegration` was ?

Reply:
the use case was because it's deleted async, so we needed to have a pending deletion status to prevent it from popping back up in the UI while it was in the process of being deleted

Reply:
Sigh... @EvanPurkhiser should I not check the status of the OrganizationIntegraiton/Integration if we are not going to keep them up to date? I think elsewhere in the code (VSTS not bitbucket) I was checking on the status there